generators/app/templates/src/app/components/Navbar/templateRoutes.js

```diff
@@ -0,0 +1,37 @@
+import ROUTES from '../../../constants/routes';
+
+import image from './example.png';
+
+// TODO customize your routes
+
+const NAVBAR_ROUTES = [
```

Comment:
I don't think we need to have this routes constant inside the bootstrap as each project may define slightly different routes to the other. The navbar might have the logic to call a route change but the constant  won't be useful in the actual projects.